music.js

```diff
@@ -1,17 +1,28 @@
 // Define the Artist constructor function here, with a 'name' property that can be set at instantiation,
 // and a 'songs' property that is an empty array to start
-function Artist() {
-
+function Artist(naam) {
+this.name = naam;
+this.song = [];
 }
 
 // Define the Song constructor function here, with 'title' and 'artist' properties that can be set at instantiation,
 // and a 'playCount' property that is an empty array to start
-function Song() {
+function Song(tl, art) {
+this.title = tl;
+this.artist = art;
+this.playCount = 0;
+
 
   // When a new song is instantiated, you'll want to add that song to its artist's 'songs' array.
+
+  Artist.prototype.newSong = function (tl) {
```

Comment:
You're doing a `prototype` method inside a function constructor, so this won't work.